### ISSUE TITLE
Úprava gesta pro mazání

### ISSUE THREAD

**Issue by kejml** [lucien144/fyx#296]
Nemám úplně konkrétní návrh, ale už dvakrát se mi stalo, že jsem omylem smazal příspěvek v diskuzi. Používám totiž v Androidu gesta, tedy pro návrat z diskuze používám tažení prstu z okraje obrazovky do středu. A přitom se právě občas chytne i ono mazání.

Nevím úhloměr cos tím. Pár návrhů:
* Reagovat jen pokud se příspěvek chytne "uprostřed"
* Potvrzovací dialog (gesto?)
* ???
* PROFIT

**Comment by lucien144** [comment]
Souhlasím. Už jsem nad tím přemýšlel - mám v plánu udělat bulk akce opačným swipem, kdy si takhle vybereš víc příspěvků a pak budeš moci mazat nebo odpovídat hromadně. Přibyly by pak adekvátně FAB prvky pro mazání odpověď.

Přemýšlím, že by mazání buď fungovalo podobně - tzn. pouze přes hromadný výběr - tím by se muselo i FAB tlačítkem potvrzovat. Nebo bych zachoval delete swipe, ale přidal bych další prvek potvrzení - buď popup nebo by vyskočil FAB na chvíli.

Ještě přemýšlím, že by to potvrzení mohlo být volitelné. Dokážu si představit, že některým adminům ta rychlá možnost mazání bez potvrzení vyvhovuje...

**Comment by kejml** [comment]
Už jsem si zase smazal vlastní příspěvek, já si snad budu muset zatím udělat vlastní build bez tohodle gesta. Vždycky mě to vytočí, ale už s tím nic nenadělám :(

**Comment by lucien144** [comment]
Ještě chvíli počkej. Mám už v plánu řešení...

**Comment by kejml** [comment]
Nemůžu se dočkat :) jestli konkrétně s tímhle můžu jakkoli pomoct, tak to na mne hoď

**Comment by lucien144** [comment]
@kejml Zamýšlím něco takového: https://x.144.wtf/wEJMkw Samozřejmě to ještě dozná změn, ale myšlenka je jasná - místo swipe to delete bude longtap to select a následně si budeš moci vybrat z různých akcí:
- smazat
- zabanovat
- odpovědět
- ...

**Comment by kejml** [comment]
To vypadá fajn :) 

Hlavně když to nebude swipe zprava bez varování, protože to je pro mne nejčastější způsob, jak opouštím diskuzi. Dokonce jsem díky tomu našel minimálně jednu diskuzi, kde maj všichni právo mazat všechny ;)
